commit d2a70d6055c5da54d3f45805a19c37a693acd07f
Author: Kamal Mostafa <kamal@canonical.com>
Date:   Wed Jan 27 22:29:33 2016 -0800

    tools/hv: Use include/uapi with __EXPORTED_HEADERS__
    
    commit 50fe6dd10069e7c062e27f29606f6e91ea979399 upstream.
    
    Use the local uapi headers to keep in sync with "recently" added #define's
    (e.g. VSS_OP_REGISTER1).
    
    Fixes: 3eb2094c59e8 ("Adding makefile for tools/hv")
    Signed-off-by: Kamal Mostafa <kamal@canonical.com>
    Signed-off-by: K. Y. Srinivasan <kys@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/tools/hv/Makefile b/tools/hv/Makefile
index a8ab795..a8c4644 100644
--- a/tools/hv/Makefile
+++ b/tools/hv/Makefile
@@ -5,6 +5,8 @@ PTHREAD_LIBS = -lpthread
 WARNINGS = -Wall -Wextra
 CFLAGS = $(WARNINGS) -g $(PTHREAD_LIBS) $(shell getconf LFS_CFLAGS)
 
+CFLAGS += -D__EXPORTED_HEADERS__ -I../../include/uapi -I../../include
+
 all: hv_kvp_daemon hv_vss_daemon hv_fcopy_daemon
 %: %.c
 	$(CC) $(CFLAGS) -o $@ $^
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1356_d2a70d6055c5da54d3f45805a19c37a693acd07f.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1356_d2a70d6055c5da54d3f45805a19c37a693acd07f.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
